src/Microsoft.AspNet.SignalR.JS/jquery.signalR.core.js

```diff
@@ -714,15 +719,16 @@
                             return;
                         }
 
-                        if (!res.ProtocolVersion || res.ProtocolVersion !== connection.clientProtocol) {
+                        if (!res.ProtocolVersion || (connection.supportedProtocols.indexOf(res.ProtocolVersion) === -1)) {
                             protocolError = signalR._.error(signalR._.format(resources.protocolIncompatible, connection.clientProtocol, res.ProtocolVersion));
                             $(connection).triggerHandler(events.onError, [protocolError]);
```

Comment:
Is it that there is no console log or error when this error happens? It fails silently when I test locally. Am I missing any config?
![image](https://user-images.githubusercontent.com/668244/45274467-2ae81580-b4ea-11e8-95a1-a536633e69bc.png)

Reply:
Yeah, we should put a log in, good catch :). I'll add that in before merging.

Reply:
Actually there are already logs, but `connection.log` writes to `console.debug` which means it only shows up if "Verbose" messages are enabled:

![image](https://user-images.githubusercontent.com/7574/45324908-5d067100-b504-11e8-8959-712f9c593677.png)

Reply:
got it! Thanks for the explanation